services/crypto_update.proto

```diff
@@ -29,6 +29,7 @@ import "basic_types.proto";
 import "duration.proto";
 import "timestamp.proto";
 import "google/protobuf/wrappers.proto";
+import "transaction_record.proto";
```

Comment:
⚠️ Unneeded import, and causes circular import:
```
[ERROR] /Users/michaeltinker/Dev/hedera-protobufs-java/src/main/proto/freeze_service.proto [0:0]: query_header.proto:28:1: File recursively imports itself: query_header.proto -> transaction.proto -> transaction_body.proto -> crypto_update.proto -> transaction_record.proto -> contract_call_local.proto -> query_header.proto
```

Reply:
Removed